components/marketing_signup_modal/index.jade

```diff
@@ -12,3 +12,4 @@ section.marketing-signup-modal
       | Continue with Facebook
     p Already have an account?&nbsp;
       a( href='/log_in' ) Log in
+    p.photo-credit= sd.MARKETING_SIGNUP_MODAL_PHOTO_CREDIT
```

Comment:
Ideally, this would be namespaced by the component `.marketing-signup-modal-photo-credit`. Not a blocker for me, but just a reminder to always namespace CSS.